### ISSUE TITLE
request parameter in beforeSave() doesn't contain any stored values sent from android app

### ISSUE THREAD

**Issue by mik-dass** [ParsePlatform/Parse-SDK-Android#166]
This is my android code :

    ParseObject parseObject = new ParseObject("likes");
                ParseObject parseuser=ParseObject.createWithoutData(ParseUser.class,post.getPosAuthorObjectId());
                ParseObject parsepost=ParseObject.createWithoutData("posts",post.getObjectId());
                parsepost.put("postAuthor",parseuser);
                parseObject.put("ownerId", ParseObject.createWithoutData(ParseUser.class, ParseUser.getCurrentUser().getObjectId()));
                parseObject.put("postId",parsepost);
                parseObject.saveInBackground();

**I am trying to send some data regarding the user's id along with the post object so as to save the post object into the "likes" class and do changes in the post owner's attributes using masterkey without fetching the id of the post owner in a separate query as it is already stored in my android code .**In my experimental beforeSave():

    Parse.Cloud.beforeSave("likes",function(request,response){
    var posts = Parse.Object.extend("posts");
    var post=new posts();
    var post=request.object.get("postId");
    var user=Parse.Object.extend(Parse.User);
    var user=new Parse.User();
    var  id =post.object.get("postAuthor").id;
    if(id!=null){
        response.success();
    }
    else{
        response.error("The id is null");
    }
     });

Basically I am trying to retrieve the user id saved in the postobject and sent to the request parameter .But the output of my experiment is always null.

**Comment by mik-dass** [comment]
@wangmengyan95 I tried to go through the documentation at the Parse website and did not find any proper explanation as to why the above code is failing. Even stack overflow doesn't contain contain any answers regarding this issue.Please help. Thank you.
